probe/kubernetes/applicationpod.go

```diff
@@ -52,6 +58,9 @@ func NewApp(p *apiv1.Pod) ApplicationPod {
 }
 
 func (p *applicationPod) GetApp(probeID string) report.Node {
+	var a annotation
+	store := []byte(p.Annotations["kubectl.kubernetes.io/last-applied-configuration"])
```

Comment:
@chandankumar4  annotation `kubectl.kubernetes.io/last-applied-configuration` will be there always?